css/_filmstrip.scss

```diff
@@ -60,6 +60,7 @@
         font-size: 0pt; /*!!!Removes the gap between the local video container and the remote videos.*/
 
         &.hidden {
+            display: block; // TOFIX: overwrites aui styles
```

Comment:
Could you please give me a bit more context about this change? Is it necessary because of React work being done in this PR? Or is it a separate fix unrelated to the React effort that you happened to discover while working?